1 HW_Standard/HW_Kabutops.yaml

```diff
@@ -19,4 +19,27 @@ shiny: True
 ivs: {hp: 31, atk: 31, def: 0, spA: 31, spD: 31, spe: 31}
 evs: {hp: 0, atk: 200, def: 0, spA: 0, spD: 72, spe: 0}
 nature: Serious
-hidden: false
\ No newline at end of file
+hidden: false
+---
+species: Kabutops
+ingamename: SCP-049
+displayname: SCP-049
+setname: Halloween
+ball: [Dusk]
+tags: [halloween]
+item: Hyper Potion
+ability: Effect Spore
+moves:
+    - [Psycho Cut]
+    - [Heal Bell]
+    - [Recover]
+    - [Fling]
```

Comment:
Potentially cancerous vs darks (can't hit them besides the pathetic resisted fling, but can recover spam and heal status)